src/gmpints.c

```diff
@@ -481,21 +567,18 @@ void PrintInt ( Obj op )
     CHECK_INT(op);
 
     /* use gmp func to print int to buffer                                 */
-    if (!IS_INTPOS(op)) {
-      buf[0] ='-';
-      signlength = 1;
-    } else {
-      signlength = 0;
-    }
-    gmp_snprintf((char *)(buf+signlength),20000-signlength,
-                 "%Nd", ADDR_INT(op),
-                 (TypGMPSize)SIZE_INT(op));
+    Char buf[20000];
+    mpz_t v;
```

Comment:
Not sure what you mean with "checking for overflows". You mean: check whether the buffer was exceeded (and we thus will crash momentarily, for smashing the stack) ? The old code had an assert for that, sure.

But given that `LogInt( (2^64)^1000, 10) = 19265 < 20000`, that simply cannot happen. Hmm, unless we run this on a 128 bit processor, I guess.